ydb/docs/en/core/dev/troubleshooting/performance/system/tablets-moved.md

```diff
@@ -0,0 +1,76 @@
+# Frequent tablet transfers between nodes
+
+{{ ydb-short-name }} automatically balances the load by transferring tablets from overloaded nodes to other nodes. This process is managed by [Hive](*hive). When Hive moves tablets, queries to those tablets suffer higher latencies.
+
+<!-- This information is taken from a draft topic Concepts > Hive. -->
+<!-- TODO: When the above-mentioned topic is merged, remove the info from here and add a link.  -->
+YDB considers usage of the following hardware resources for balancing nodes:
+
+- CPU
+- Memory
+- Network
+- [Counter](*counter)
+
+Autobalancing occurs in the following cases:
+
+- **Disbalance in hardware resource usage**
+
+    {{ ydb-short-name }} uses the **scatter** metric to evaluate the balance in hardware resource usage. This metric is calculated for each resource using the following formula:
+
+    $Scatter = \frac {MaxUsage - MinUsage} {MaxUsage},$
+
+    where:
+
+    - $MaxUsage$ is the maximum hardware resource usage among all of the nodes.
+    - $MinUsage$ is the minimum hardware resource usage among all of the nodes.
+
+    To distribute the load, {{ ydb-short-name }} takes into account the amount of hardware resources availables to each node. The scatter value might have differred greatly across the nodes under low loads, so minimum value for this formula is never below `30%`.
+
+- **Overloaded nodes (CPU and memory usage)**
+
+    Hive starts the autobalancing procesure when the highest load on a node exceeds `90%`, while the lowest load on a node is below `70%`.
+
+- **Uneven distribution of database objects**
+
+    {{ ydb-short-name }} uses the **ObjectImbalance** metric to monitor the distribution of tablets that use the **[counter](*counter)** resource across {{ ydb-short-name }} nodes. When {{ ydb-short-name }} nodes restart, these tablets might fail to distribute evenly, and Hive will start the autobalancing procedure.
+
+
+## Diagnostics
+
+<!-- The include is added to allow partial overrides in overlays  -->
+{% include notitle [#](_includes/tablets-moved.md) %}
+
+## Recommendations
+
+Adjust Hive balancer values:
+
+1. Open [Embedded UI](../../../../reference/embedded-ui/index.md).
+
+1. Click **Developer UI** in the upper right corner of the Embedded UI.
+
+1. In the **Developer UI**, navigate to **Tablets > Hive > App**.
+
+    ![](_assets/hive-app.png)
+
+1. Click **Settings**.
+
+1. To reduce the chances of balancing, increase the following Hive balancer thresholds:
+
+    - MinCounterScatterToBalance
+    - MinCPUScatterToBalance
+    - MinMemoryScatterToBalance
+    - MinNetworkScatterToBalance
+    - MaxNodeUsageToKick
+    - ObjectImbalanceToBalance
```

Comment:
This better be a table with some description, defaults, allowed and practical ranges, etc.